run-vmtest/run.sh

```diff
@@ -3,14 +3,28 @@
 set -euo pipefail
 trap 'exit 2' ERR
 
-source $(cd $(dirname $0) && pwd)/../helpers.sh
+source "${GITHUB_ACTION_PATH}/../helpers.sh"
```

Comment:
why the change from `$(cd $(dirname $0) && pwd)` to `${GITHUB_ACTION_PATH}`?

Reply:
I've been trying to use common env vars for paths everywhere, and `$GITHUB_ACTION_PATH` makes most sense for actions, when you're looking for relative paths.

However, in this particular case I don't think important, so I don't mind leaving pwd.

Any particular reasons in favor of `$(cd $(dirname $0) && pwd)`?